evm_arithmetization/src/cpu/kernel/asm/core/create.asm

```diff
@@ -211,22 +211,27 @@ nonce_overflow:
     %stack (sender, address, value, code_offset, code_len, kexit_info) -> (kexit_info, 0)
     EXIT_KERNEL
 
+// stack: new_ctx, address, kexit_info
 create_collision:
     %revert_checkpoint
     %mstore_context_metadata(@CTX_METADATA_RETURNDATA_SIZE, 0)
-    %stack (new_ctx, address, kexit_info) -> (kexit_info, 0)
+    %prune_context
+    %stack (address, kexit_info) -> (kexit_info, 0)
```

Comment:
minor: would deserve some comment clarification that collision are checked when running the constructor, _prior_ jumping to the new context, hence the pruning (unlike the other checks below that are run _after_)

Reply:
@LindaGuiga can we get this addressed and merged soon? This is blocking @atanmarko and his benchmarking PR (some segments in the targeted artifact are suffering from the bug this PR fixes).